plugins/auth-backend-module-oidc-proxy-provider/README.md

```diff
@@ -0,0 +1,6 @@
+# Auth Backend Module - OIDC Proxy Provider
+
+## Links
+
+- [The Backstage homepage](https://backstage.io)
+- [Open Infrastructure Services LLC](https://openinfrastructure.co) - Original authors of this provider. Maintainers of Holos, which this provider was originally purpose built for.
```

Comment:
probably link to the new docs that this PR creates, so people can easily find how to install it

Reply:
Linked to https://backstage.io/docs/auth/oidc/provider/